### ISSUE TITLE
Contacts Birthdays

### ISSUE THREAD

**Issue by syborgxx** [39aldo39/DecSyncCC#67]
Since DecSync is managing contacts, a calendar with contacts birthdays should be generated by DecSync.

**Comment by 39aldo39** [comment]
I don't think it is necessarily the responsibility of DecSync. For example, Evolution already creates such calendar for all contacts. On Android this is not done by default, but there are apps like [Birthday Adapter](https://f-droid.org/en/packages/org.birthdayadapter/) that accomplish it.
